### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by haliljon**
## In this PR I was able to:

> - use gitflow
> - use proper ES6 syntax.
> - use ES6 modules to write modular JavaScript.
> - use webpack to bundle JavaScript.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
```
Comment (christianbiring1) [inline package.json]: - [ ] Kindly make sure you have all the needed scripts to run the project locally with the webpack bundler.
`HInt: YOu may consider having these script`
```json
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
```

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @haliljon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
almost


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
- [ ] `KINDLY MAKE SURE YOUR WEB PACK CONFIGURATION IS SET UP CORRECTLY and in your package.json KINDLY MAKE SURE WEHAVE ALL THE NEEDED DEPENDENCIES`
Screenshot from 2023-01-04 23-30-03
- [ ] `kindly MAKE SURE YOUR WORK IS DOCUMENTED IN A PROFESSIONAL WAY, YOU MAY CONSIDER USING THE TEMPLATE PROVIDED TO YOU AND PERSONALIZE TO WHAT APPLIES ON YOUR PROJECT` [here](https://github.com/microverseinc/readme-template)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (haliljon) [reply package.json]: Hi @christianbiring1, thank you for the review

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @haliljon,

✔️ App is set up with webpack

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**